index.bs

```diff
@@ -314,7 +315,7 @@ communication methods.
 
 When a specification exposes specific information about a host to an origin,
 if that information changes rarely and is not variable across origins, then
-it can be used to uniquely identify a user across two origins — either
+it can be used to uniquely identify a user's agent across two origins — either
```

Comment:
Firstly it should specifically be talking about user-agent (vs user's agent) as that is a term of art. Secondly, in some cases we could be talking about users and in some cases user agents - as for example when a unqiue identifier could be accessible to two user agents.

Reply:
Resolved in latest commit.